README.md

```diff
@@ -291,13 +284,13 @@ Bazel label names (e.g. `-`, `.`) replaced with `_`. If you need to
 update `name` from "old" to "new", then you can run the following
 buildozer command:
 
-```
+```bash
```

Comment:
nit: shell is probably a more appropriate name than bash, but i'm not entirely certain on what flavors of language names are understood by github's markdown.